stable/stolon/templates/configmap.yaml

```diff
@@ -30,3 +30,6 @@ data:
 
     echo "Node ${NODE_NAME} is ready to shutdown"
 {{- end }}
+{{- with .Values.nodePostStartScript }}
+{{ toYaml . | indent 2 }}
+{{ end }}
```

Comment:
`{{- end}}` can make sense more than `{{ end }}`

Reply:
Please can you explain me the meaning of "-".

Reply:
Fixed